sccm/core/plan-design/hierarchy/fundamental-concepts-for-content-management.md

```diff
@@ -52,7 +52,7 @@ For more information, see [Package access account](/sccm/core/plan-design/hierar
 
 
 ## Binary differential replication  
- Binary differential replication (BDR) is a prerequisite for distribution points. It's sometimes known as delta replication. When you're distributing updates to content that you previously deployed to other sites or to remote distribution points, BDR is automatically used to reduce bandwidth.  
+ Binary differential replication (BDR), sometimes known as delta replication, is used to distribute updates to content that you previously deployed to other sites or to remote distribution points. Distribution points [require](https://docs.microsoft.com/en-us/sccm/core/plan-design/configs/site-and-site-system-prerequisites#bkmk_2012dppreq) the Remote Differential Compression feature to be installed to support BDR's reduction of bandwidth usage.

```

Comment:
A few editorial suggestions:
1. [Shorter is always better](https://docs.microsoft.com/style-guide/top-10-tips-style-voice#be-brief). This style recommendation is for clarity but also better localization in other languages.
2. When adding links, always use language neutral links by removing the language code. For links internal to this repo, just use a relative link. 
3. Generally (not always), refer to additional content in a separate sentence: "For more information, see Article title."

```suggestion
 Binary differential replication (BDR) is sometimes known as delta replication. It's used to distribute updates to content that you previously deployed to other sites or to remote distribution points. To support BDR's reduction of bandwidth usage, install the **Remote Differential Compression** feature on distribution points. For more information, see [Distribution point prerequisites](/sccm/core/plan-design/configs/site-and-site-system-prerequisites#bkmk_2012dppreq).
```